sermant-plugins/sermant-mq-grayscale/mq-grayscale-rocketmq-plugin/src/main/java/io/sermant/mq/grayscale/rocketmq/utils/RocketMqGrayscaleConfigUtils.java

```diff
@@ -72,7 +76,16 @@ public static String getGrayGroupTag() {
         }
         Optional<GrayTagItem> itemOptional
                 = MqGrayConfigCache.getCacheConfig().getMatchedGrayTagByServiceMeta(MICRO_SERVICE_PROPERTIES);
-        return itemOptional.map(grayTagItem -> standardFormatGroupTag(grayTagItem.getConsumerGroupTag())).orElse("");
+        if (itemOptional.isPresent()) {
+            String consumerGroup = itemOptional.get().getConsumerGroupTag();
+            if (PATTERN.matcher(consumerGroup).matches()) {
+                return itemOptional.get().getConsumerGroupTag();
```

Comment:
itemOptional.get() can be define as a variable

Reply:
fixed